pro/src/core/shared/utils/validation.ts

```diff
@@ -1,10 +1,10 @@
-import { parsePhoneNumberFromString } from 'libphonenumber-js'
+import { parseAndValidateFrenchPhoneNumber } from './parseAndValidateFrenchPhoneNumber'
 
 export const isPhoneValid = (phone: string | undefined): boolean => {
   if (!phone) {
     return false
   }
-  const phoneNumber = parsePhoneNumberFromString(phone, 'FR')
+  const phoneNumber = parseAndValidateFrenchPhoneNumber(phone)
```

Comment:
On fait déjà un .isValid dans le parseAndValidateFrenchPhoneNumber, de ce fait on pourrait faire : 

```suggestion
  try { 
      parseAndValidateFrenchPhoneNumber(phone)
  }
  catch { return false }
  return true
```